src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -271,44 +296,12 @@ struct EvolutionMetavars {
       tmpl::conditional_t<use_dg_subcell, dg_subcell_step_actions,
                           dg_step_actions>;
 
-  enum class Phase {
-    Initialization,
-    LoadBalancing,
-    WriteCheckpoint,
-    RegisterWithObserver,
-    InitializeTimeStepperHistory,
-    Evolve,
-    Exit
-  };
-
-  static std::string phase_name(Phase phase) {
-    if (phase == Phase::LoadBalancing) {
-      return "LoadBalancing";
-    }
-    ERROR(
-        "Passed phase that should not be used in input file. Integer "
-        "corresponding to phase is: "
-        << static_cast<int>(phase));
-  }
-
-  using phase_changes =
-      tmpl::list<PhaseControl::Registrars::VisitAndReturn<EvolutionMetavars,
-                                                          Phase::LoadBalancing>,
-                 PhaseControl::Registrars::CheckpointAndExitAfterWallclock<
-                     EvolutionMetavars>>;
-
-  using initialize_phase_change_decision_data =
-      PhaseControl::InitializePhaseChangeDecisionData<phase_changes>;
-
-  using phase_change_tags_and_combines_list =
-      PhaseControl::get_phase_change_tags<phase_changes>;
-
   using const_global_cache_tags = tmpl::list<
       initial_data_tag, Tags::EventsAndTriggers,
       tmpl::conditional_t<use_dg_subcell,
                           tmpl::list<Burgers::fd::Tags::Reconstructor>,
                           tmpl::list<>>,
-      PhaseControl::Tags::PhaseChangeAndTriggers<phase_changes>>;
+      PhaseControl::Tags::PhaseChangeAndTriggers>;
```

Comment:
We collect cache tags from actions, right? In that case I think you can list this tag in the ExecutePhaseChange action and delete it here. Same for the EventsAndTriggers actually I think.

Reply:
Good idea.  Done both.